diff --git a/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js b/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
index 47487c0..486eb6a 100644
--- a/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
+++ b/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
@@ -137,7 +137,7 @@ ${generateReactDelegateHandlers(reactDelegateHandlerModules, debugOnlyReactDeleg
 `;
 }
 function generateCommonImportList(swiftModules) {
-    return swiftModules.map((moduleName) => `import ${moduleName}`).join('\n');
+    return swiftModules.map((moduleName) => moduleName === 'Expo' ? `internal import ${moduleName}` : `import ${moduleName}`).join('\n');
 }
 function generateDebugOnlyImportList(swiftModules) {
     if (!swiftModules.length) {
